modules/installation-arm-bootstrap.adoc

```diff
@@ -0,0 +1,257 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-user-infra.adoc
+
+[id="installation-arm-bootstrap_{context}"]
+= ARM template for the bootstrap machine
+
+You can use the following Azure Resource Manager (ARM) template to deploy the
+bootstrap machine that you need for your {product-title} cluster:
+
+.`04_bootstrap.json` ARM template
+[source,json]
```

Comment:
Is it possible, in the docs engine you use, to import the content from external files? For example, `04_bootstrap.json` in the 4.4 branch is located in the URL below. I'm just a little bit worried about us eventually backporting a bugfix, for example, and this not being reflected in docs.

https://raw.githubusercontent.com/openshift/installer/release-4.4/upi/azure/04_bootstrap.json

Reply:
We don't have anything like that currently in our docs, but I can further research this for future use. After some exploring, it is possible with Asciidoc, but I'm not sure if our current docs would support it. In the past, I've used something like [this](https://app.github-file-watcher.com/) for cases where we should monitor a doc for changes, but that's obviously not ideal in the long term.

Reply:
@vikram-redhat Any updates on how to handle big ol'files like this? The above suggestion is definitely possible with adoc passthroughs, though I don't know if that method is allowed in our docs.

Reply:
Hey @maxwelldb - no, nothing yet. I think the suggestion from @fabianofranz is sound, but even if we were able to find out a way to inject it, I think it will need to be accompanied by the review process, especially QE acks. That is why hosting of these files by us is problematic.

@sferich888 - would you be ok with us linking to these files for installer on GitHub? OR if we were to host them on docs.openshift.com in a raw format (ex: docs.openshift.com/file/<filename>) and updated periodically with updates to them vetted by the team before publishing?

Reply:
@vikram-redhat Thanks for checking! 

We should look at having somewhere to keep QE-approved scripts and files. Having to embed them as blocks in our docs seems really suboptimal for usability and maintenance.

Reply:
I agree with @maxwelldb that embedding into our content creates a poor end-user experience.